src/client/App.js

```diff
@@ -43,8 +42,8 @@ function App() {
             <Route exact path="/selection">
               <SelectionPage />
             </Route>
-            <Route exact path="/registration/student">
-              <StudentRegistrationPage />
+            <Route exact path="/registration/mentor">
+              <MentorRegistrationPage />
```

Comment:
Same as in the other PR, you removed student registration.

Reply:
updated